admin/app/components/solidus_admin/adjustment_reasons/edit/component.html.erb

```diff
@@ -1,6 +1,6 @@
-<%= turbo_frame_tag :edit_adjustment_reason_modal do %>
-  <%= render component("ui/modal").new(title: t(".title")) do |modal| %>
-    <%= form_for @adjustment_reason, url: solidus_admin.adjustment_reason_path(@adjustment_reason), html: { id: form_id } do |f| %>
+<%= turbo_frame_tag :edit_adjustment_reason_modal, target: "_top" do %>
+  <%= render component("ui/modal").new(title: t(".title"), close_path: close_path) do |modal| %>
+    <%= form_for @adjustment_reason, url: solidus_admin.adjustment_reason_path(@adjustment_reason, **search_filter_params), html: { id: form_id } do |f| %>
```

Comment:
Nice this is exactly what I was hoping for

Reply:
Fun fact: We already do that in the pagination. Just making it a bit safer, by filtering only the params we need and pass this around with all links.

https://github.com/solidusio/solidus/blob/dcb0dd820a7699a4676a257c3363e747eb09b549/admin/app/components/solidus_admin/ui/pages/index/component.rb#L48-L54